### PR TITLE
add scheduled test-run at night

### PR DESCRIPTION
**Pull request by syphar**
following the issue with nightly breaking our build we should start having a nightly test-run to be notified when the build breaks. 

IMHO it doesn't hurt to just run the normal CI-job, of course we could split these into two separate workflows where we only schedule the build-test.

### CODE REVIEW

**Comment by syphar** [comment]
IMHO we can block this until the build-tests are green again.

**Review by jyn514 — APPROVED** [review]
Not sure how to make sure this works, but I guess we'll see once it's merged and the checks start running :laughing:

**Comment by syphar** [comment]
> Not sure how to make sure this works, but I guess we'll see once it's merged and the checks start running 😆

you can see it working similarly in https://github.com/rust-lang/docs.rs/blob/4ef69ca935088bb6b5d2df9ae1ad95b16d410a40/.github/workflows/audit.yml#L3-L10

**Pull request merged**
